packages/auth/src/index.ts

```diff
@@ -1,6 +1,6 @@
 import { EventEmitter } from 'events'
 import createLogger from 'pino'
-import Knex from 'knex'
+import { knex } from 'knex'
```

Comment:
Should it be uppercase `Knex` here?

Reply:
Ditto: https://github.com/knex/knex/blob/master/UPGRADING.md#upgrading-to-version-0950